setup.py

```diff
@@ -10,10 +10,19 @@
         "hypothesis>=6.2.0,<7.0",  # Strategy-based fuzzer
     ],
     "lint": [
-        "black>=22.6.0",  # auto-formatter and linter
-        "mypy==0.982",  # Static type analyzer
-        "flake8>=4.0.1",  # Style linter
+        "black>=22.12.0",  # auto-formatter and linter
+        "mypy>=0.991",  # Static type analyzer
+        "types-setuptools",  # Needed for mypy type shed
+        "flake8>=5.0.4",  # Style linter
         "isort>=5.10.1",  # Import sorting linter
+        "mdformat>=0.7.16",  # Auto-formatter for markdown
```

Comment:
There are couple small changes to mdformat here: https://github.com/ApeWorX/project-template/pulls
Just adding the config file mostly.